spec/models/book_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe Book do
+  let(:book) { build(:book) }
+
+  context 'When creates a book' do
```

Comment:
generally we don't add a context when the validations apply to every instance of an entity. In this case you could remove the context.